### ISSUE TITLE
Headers regularly collapse int the search windows

### ISSUE THREAD

**Issue by Npa972** [DeaDBeeF-Player/deadbeef#2963]
### Steps to reproduce the problem
Ctrl + F

### What's going on? Describe the problem in as much detail as possible.
Headers collapse regularly. It seems to appear after closing and restarting Deadbeef

### Information about the software:
Deadbeef version: devel version as well as 1.9.5

OS: arch

![Capture d’écran_2023-07-01_10-22-16](https://github.com/DeaDBeeF-Player/deadbeef/assets/49346431/6adb0f02-b245-4fb9-bd66-ae610c223b98)
![Capture d’écran_2023-07-01_10-24-11](https://github.com/DeaDBeeF-Player/deadbeef/assets/49346431/aa826e2f-dfa2-420b-9ee2-dec7fff0b301)

Thanks

**Comment by Oleksiy-Yakovenko** [comment]
Hi,

is it something that started to occur recently? perhaps after an Arch update?
Also, does it happen in the playlist header in the main window?

**Comment by kpcee** [comment]
I had the same problem a long time ago, but since I turned off the option "GUI/Misc → Playlist → Auto resize columns to fit the window", there were no more problems. If I remember correctly, only the search window was affected.

**Comment by Npa972** [comment]
@Oleksiy-Yakovenko yes only in the search windows. Playlist is not affected. Arch update, i don't know.
But i use auto resize, good option to resize the main window. Perhaps there is something there.
As proposed by @kpcee I turn it off to see what's happen.

**Comment by Npa972** [comment]
No problem since i turn off auto-resize columns.
The search windows was affected (all columns completely collapse) but sometimes i had to readjust columns width a little in the playlist to fit the screen. This is also solved with auto-resize off.

**Comment by Oleksiy-Yakovenko** [comment]
Duplicate of #3199
